File: app/src/main/java/com/example/sergio/todo_list_firebase/models/Priority.java
package com.example.sergio.todo_list_firebase.models;

/**
 * Created by sergio on 23/04/2016.
 */
public class Priority {

    private int id;
    private String name;
    private String desc;

    public Priority(){}

    public Priority(int id, String name, String desc) {
        this.id = id;
        this.name = name;
        this.desc = desc;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getDesc() {
        return desc;
    }

    public void setDesc(String desc) {
        this.desc = desc;
    }
}

File: app/src/main/java/com/example/sergio/todo_list_firebase/domain/models/Priority.kt
package com.example.sergio.todo_list_firebase.domain.models

/**
 * Created by sergio on 23/04/2016.
 */
class Priority {

    var id: Int = 0
    var name: String? = null
    var desc: String? = null

    constructor() {
    }

    constructor(id: Int, name: String, desc: String) {
        this.id = id
        this.name = name
        this.desc = desc
    }
}
